src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx

```diff
@@ -17,16 +17,20 @@ If you don't have one already, [create a New Relic account](https://newrelic.com
 
```

Comment:
Needs format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'